src/api/eventTypes.js

```diff
@@ -100,23 +100,54 @@ export type PresenceEvent = $ReadOnly<{|
 |}>;
 
 /**
- * Updates the user status for a user
+ * New value for a user's self-assigned availability and/or text/emoji
+ *   status
  *
- * @prop [away] - update user's away status:
- *       - `true` the user is away regardless of presence
- *       - `false` remove the away status, now use presence
- * @prop [status_text] - if present:
- *       - empty string clears the user's status text
- *       - any string sets user's status to that
+ * See InitialDataUserStatus for the corresponding initial data. The
+ * per-user properties have different meanings between here and there.
```

Comment:
Let's also link to the API docs:
  https://zulip.com/api/get-events#user_status